ru.json

```diff
@@ -1,5 +1,5 @@
 {
-    "lng_intro_about2": "Добро пожаловать в приложение 64Gram Desktop.\nОно быстрое и безопасное.",
+    "lng_intro_about2": "Добро пожаловать в приложение 64Gram Desktop. / nОно быстрое и безопасное.",
```

Comment:
Why you changing it?
`\n` is a newline character.

Reply:
for the Russian language it looks better)